packages/core/src/scss/components/bigcommerce/loading/_loading.scss

```diff
@@ -64,3 +64,13 @@
         width: 0;
     }
 }
+
+.embedded-checkout-loading-spinner-overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    background-color: rgba(1, 1, 1, .3);
```

Comment:
Could you please move the colour code to the SCSS setting?
https://github.com/bigcommerce/checkout-js/blob/master/packages/core/src/scss/settings/global/overlay/_overlay.scss

Reply:
moved

Reply:
Please just move `rgba(1, 1, 1, .3)`  so that we can reuse this colour as a setting value.